lib-rust/kson/src/lib.rs

```diff
@@ -185,6 +180,40 @@ declare_kotlin_object! {
     Position
 }
 
+impl std::fmt::Debug for Position {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Position")
+            .field("line", &self.line())
+            .field("column", &self.column())
+            .finish()
+    }
+}
+
+impl PartialEq for Position {
+    fn eq(&self, other: &Self) -> bool {
+        self.line() == other.line() && self.column() == other.column()
```

Comment:
Is there a reason to have this logic in Rust, instead of calling into Kotlin here? If Kotlin's position has an equals method similar to this one, we should call it instead of reimplementing it.